src/transform-sdk/js/CMakeLists.txt

```diff
@@ -65,13 +65,14 @@ target_link_libraries(
   qjs Redpanda::transform_sdk Redpanda::js_vm
 )
 
-if(CMAKE_BUILD_TYPE MATCHES Release)
-  include(CheckIPOSupported)
-  check_ipo_supported(RESULT ltosupported OUTPUT error)
-  if(ltosupported)
-    set_property(TARGET redpanda_js_transform PROPERTY INTERPROCEDURAL_OPTIMIZATION ON)
-  endif()
-endif()
+# TODO: Rename LTO builds when #18077 is merged
```

Comment:
#18077 has merged, re-enable?

Reply:
Yes will do in a followup, I would like to get this in for now